src/renderer/library/components/searchResult/AllPublicationPage.tsx

```diff
@@ -1478,6 +1519,9 @@ export const TableView: React.FC<ITableCellProps_TableView & ITableCellProps_Com
                     label: publicationView.authors ? publicationView.authors.join(", ") : "",
                     authors: publicationView.authors,
                 },
+                colReadingState: { // IColumnValue_Authors
+                    label: publicationView.readingFinished ? `${props.__("publication.read")}` : publicationView.lastReadingLocation ? `${props.__("publication.onGoing")}` : "",
+                },
```

Comment:
See https://github.com/edrlab/thorium-reader/issues/2110
I propose "Finished" (instead of "Read" which is both a verb and an adjective so can be confused for an action indicator / affordance instead of a state)
I propose "In Progress" instead of "On Going" (which is incorrect anyway, should be "Ongoing")
I propose "Not Started" instead of empty string so that the column cells are sorted and batched together when the user looks for publications that are not opened yet.